code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm

```diff
@@ -142,35 +160,34 @@
 	description = "Used to treat burns. Makes you move slower while it is in your system. Applies stomach damage when it leaves your system."
 	reagent_state = LIQUID
 	color = "#6171FF"
-	var/resetting_probability = 0
+	ph = 4.7
+	impure_chem = /datum/reagent/impurity/lentslurri
+	failed_chem = /datum/reagent/inverse/ichiyuri //I do hope cobby won't take this personally
+	var/resetting_probability = 0 //What are these for?? Can I remove them?
 	var/spammer = 0
 	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED
 
 /datum/reagent/medicine/c2/lenturi/on_mob_life(mob/living/carbon/M, delta_time, times_fired)
-	M.adjustFireLoss(-3 * REM * delta_time)
+	M.adjustFireLoss(-3 * REM * normalise_creation_purity() * delta_time)
 	M.adjustOrganLoss(ORGAN_SLOT_STOMACH, 0.4 * REM * delta_time)
 	..()
 	return TRUE
 
-/datum/reagent/medicine/c2/lenturi/on_mob_metabolize(mob/living/carbon/M)
-	M.add_movespeed_modifier(/datum/movespeed_modifier/reagent/lenturi)
-	return ..()
-
-/datum/reagent/medicine/c2/lenturi/on_mob_end_metabolize(mob/living/carbon/M)
-	M.remove_movespeed_modifier(/datum/movespeed_modifier/reagent/lenturi)
-	return ..()
+//lenturi side effects are now part of the impure type made from it
```

Comment:
Comments shouldn't explain what code *used* to be.

Reply:
Deleted